pkg/gitreceive/build.go

```diff
@@ -183,14 +183,19 @@ func build(
 		return fmt.Errorf("creating builder pod (%s)", err)
 	}
 
-	if err := waitForPod(kubeClient, newPod.Namespace, newPod.Name, conf.SessionIdleInterval(), conf.BuilderPodTickDuration(), conf.BuilderPodWaitDuration()); err != nil {
+	pw := k8s.NewPodWatcher(kubeClient, "deis")
```

Comment:
I had done 2 git pushes simultaneously and observed the packets using [iptraf](http://iptraf.seul.org/2.2/manual.html) and could see only one packet sent per app